projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -327,6 +382,11 @@ class FakeProjectionActor(tagName: String, testProbe: TestProbe) extends Actor w
     testProbe.ref ! FakeStarting(tagName)
   }
 
+  override def postStop(): Unit = {
+    super.postStop()
+    testProbe.ref ! FakeStopping(tagName)
```

Comment:
Introduced this one so we can properly observe when it stops

Reply:
Nice addition!